sample/app/src/main/java/com/example/activitystarter/BaseActivity.java

```diff
@@ -1,12 +1,12 @@
 package com.example.activitystarter;
 
 import android.os.Bundle;
-import android.support.v7.app.AppCompatActivity;
 
 import activitystarter.ActivityStarter;
+import androidx.appcompat.app.AppCompatActivity;
 
-
-public class BaseActivity extends AppCompatActivity {
+public class BaseActivity extends AppCompatActivity
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html